packages/ember-data/lib/system/record-arrays/record-array.js

```diff
@@ -135,22 +135,6 @@ export default Ember.ArrayProxy.extend(Ember.Evented, {
     }
   },
 
-  _pushRecord: function(record) {
-    get(this, 'content').pushObject(record);
-  },
-
-  /**
-    Adds a record to the `RecordArray`, but allows duplicates
-
-    @deprecated
-    @method pushRecord
-    @private
-    @param {DS.Model} record
-  */
-  pushRecord: function(record) {
-    Ember.deprecate('Usage of `recordArray.pushRecord` is deprecated, use `recordArray.addObject` instead');
```

Comment:
but i bet this breaks with references refactor. Should call this.pushObject instead of on the content